roles/keystone/tasks/main.yml

```diff
@@ -56,6 +57,12 @@
     - 5001
     - 35358
 
+- name : change keystone log file ownership
+  file: path={{ item }} mode=0644 owner=keystone group=keystone state=file
+  with_items:
+    - /var/log/keystone/keystone-all.log
+    - /var/log/keystone/keystone-manage.log
```

Comment:
I'm leaning towards the touch method. Let me commit some stuff on another branch I'm working on and I'll update the PR.